CerebNet/inference.py

```diff
@@ -291,6 +291,7 @@ def _get_ids_startswith(_label_map: Dict[int, str], prefix: str) -> List[int]:
         table = pv_calc(
             seg_data,
             norm_data,
+            norm_data,
```

Comment:
why do we have norm_date duplicated here?

Reply:
These are the values for `norm_data` and `pv_guide`. One is used to calculate the intensity values the other to determine the pv corrected volume.

Mri_segstats uses -pv and -norm as equivalent flags.

Reply:
... And in CerebNet (here) this is actually the same data (usually also in FreeSurfer, but this is not required)